src/Controllers/ModuleSuiteLocator.php

```diff
@@ -190,4 +196,30 @@ protected function loadSuiteConfiguration($suite, Module $module)
             'settings' => $resolvedConfig,
         ];
     }
+
+    private function setI18nLocale(array $config): void
+    {
+        $path = $config['type']['settings']['paths'][0] ?? '';
```

Comment:
Explicitly only checking a single path will give unexpected results if multiple paths are included - but I *think* we only ever use a single path for our behat suites (and even if we use multiple our own suites will always be supported anyway) - and if someone else is running our tests in their own project then I think it's fine for unexpected results. Our tests aren't meant to be run in conjunction with other tests.